build.sbt

```diff
@@ -86,6 +88,14 @@ lazy val kafka =
     .dependsOn(testkit % Test)
     .dependsOn(slick % "test->test;it->it")
 
+// provides source providers for durable state changes
+lazy val state =
+  Project(id = "akka-projection-state", base = file("akka-projection-state"))
```

Comment:
I think we should call it `akka-projection-durable-state` or `akka-projection-durable-state-changes` (maybe too long). 

`akka-projection-state` may be confusing as it may give the impression that this is about the state of the projections themselves (ie: running, paused, current offset).

Reply:
+1 to using `akka-projection-durable-state`. `state` by itself is so overloaded.